lib/Dialect/AIE/Transforms/AIEObjectFifoStatefulTransform.cpp

```diff
@@ -226,9 +226,8 @@ struct AIEObjectFifoStatefulTransformPass
         BufferOp buff = builder.create<BufferOp>(builder.getUnknownLoc(),
                                                  elemType, creation_tile);
         buff.getOperation()->setAttr(
-            "sym_name",
-            builder.getStringAttr("of_" + std::to_string(of_index) + "_buff_" +
-                                  std::to_string(of_elem_index)));
+            "sym_name", builder.getStringAttr(op.name().getValue() + "_buff_" +
```

Comment:
We should be using `SymbolTable::getSymbolAttrName()` instead of `"sym_name"` here and several other places.